store/tikv/unionstore/union_store.go

```diff
@@ -139,6 +139,12 @@ func (us *KVUnionStore) GetOption(opt int) interface{} {
 	return us.opts[opt]
 }
 
+// SetEntrySizeLimit sets the size limit for each entry and total buffer.
+func (us *KVUnionStore) SetEntrySizeLimit(entryLimit, bufferLimit uint64) {
```

Comment:
So we should attention that it is ok for `tikv-client` to update the entrysizelimit in a transaction.